### PR TITLE
Update for Quarkus 2.0.0.Final

### PR DESCRIPTION
**Pull request by jamesfalkner**
@edeandrea please review. There is a corresponding PR https://github.com/openshift-katacoda/rhoar-getting-started/pull/78 that must merge along with this.

### CODE REVIEW

File: middleware/middleware-quarkus/getting-started/01-create-project.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ First, change to the directory in which the project was created:
 
 Now we are ready to run our application. Click here to run:
 
-```mvn compile quarkus:dev -Dquarkus.http.host=0.0.0.0```{{execute}}
+```mvn quarkus:dev -Dquarkus.http.host=0.0.0.0```{{execute}}
```
Comment (edeandrea) [inline middleware/middleware-quarkus/getting-started/01-create-project.md:93]: Why did you remove `compile` from the command? I was told we should always include it. Just running `quarkus:dev` does not run the `generate-code` goal.

File: middleware/middleware-quarkus/getting-started/01-create-project.md [hunk 2]
```diff
@@ -134,8 +136,28 @@ For example, click on the `Config Editor` link within the `Configuration` tile t
...
+```
+All 1 tests are passing (0 skipped), 1 tests were run in 389ms. Tests completed at 12:25:40 due to changes to GreetingResource.class.
+```
+
+Quarkus analyses your unit tests and only re-runs the tests that are affected by code changes. It's one of many developer productivity features of Quarkus, providing immediate feedback to developers as they type. We'll leave the tests running continously just to ensure we don't mess up later.
+
 # Congratulations!
 
-You've seen how to build a basic app, package it as an executable JAR and start it up very quickly. We'll leave the app running and rely on hot reload for the next steps.
+You've seen how to build a basic app, package it as an executable JAR and start it up very quickly. You also saw how Quarkus can run tests continuously to turbocharge your development tasks. We'll leave the app running and rely on hot reload for the next steps.
```
Comment (edeandrea) [inline middleware/middleware-quarkus/getting-started/01-create-project.md:161]: What about also adding something to this sentence like "... turbocharger your development tasks and facilitate test-driven development."

?

File: middleware/middleware-quarkus/getting-started/03-package-and-run.md [hunk 3]
```diff
@@ -31,8 +31,9 @@ And then test it again using the browser to access the `/hello/greeting` endpoin
 You should see:
 
 ```console
-hello quarkus from master
+hello quarkus from 4090f59d1a69
 ```
+Note the same hostname as before, and also note that since we're running it as an Uber JAR, Quarkus runs in production mode (and does not enable continuous testing or other developer features).
```
Comment (edeandrea) [inline middleware/middleware-quarkus/getting-started/03-package-and-run.md:36]: But we aren't running it as an uber-jar. We're running it as a fast jar!

**Review by edeandrea — COMMENTED** [review]
@jamesfalkner - Comments for the `getting-started` scenario in-line.

File: middleware/middleware-quarkus/kafka/04-create-resource.md [hunk 4]
```diff
@@ -17,6 +17,7 @@ import javax.ws.rs.GET;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
+import org.jboss.resteasy.annotations.SseElementType;
```
Comment (edeandrea) [inline middleware/middleware-quarkus/kafka/04-create-resource.md:20]: If we convert to use RESTEasy reactive then this isn't needed.

File: middleware/middleware-quarkus/kafka/04-create-resource.md [hunk 5]
```diff
@@ -30,6 +31,7 @@ public class NameResource {
     @GET
     @Path("/stream")
     @Produces(MediaType.SERVER_SENT_EVENTS)
+    @SseElementType("text/plain")
```
Comment (edeandrea) [inline middleware/middleware-quarkus/kafka/04-create-resource.md:34]: If we convert to use RESTEasy reactive then this isn't needed.

File: middleware/middleware-quarkus/kafka/04-create-resource.md [hunk 6]
```diff
@@ -39,9 +41,8 @@ public class NameResource {
 This method:
 
   - `@Inject`s the `my-data-stream` stream using the `@Channel` qualifier
-
   - Indicates that the content is sent (`@Produces`) using *Server Sent Events*
-
+  - Indicates that the data contained within the server sent events is of type `text/plain`
```
Comment (edeandrea) [inline middleware/middleware-quarkus/kafka/04-create-resource.md:45]: If we convert to use RESTEasy reactive then this isn't needed.

**Review by edeandrea — COMMENTED** [review]
@jamesfalkner Comments re: the kafka scenario

File: middleware/middleware-quarkus/monitoring/01-install-prometheus.md [hunk 7]
```diff
@@ -49,21 +49,20 @@ scrape_configs:
...
 Next, click this command to create a ConfigMap with the above file:
 
 `oc create configmap prom --from-file=prometheus.yml=/tmp/prometheus.yml`{{execute}}
 
 ## Deploy Prometheus
 
-Next, deploy and expose Prometheus using its public Docker Hub image:
+Next, deploy and expose Prometheus using its public Quay.io image:
 
 `oc new-app quay.io/prometheus/prometheus && oc expose svc/prometheus`{{execute}}
```
Comment (edeandrea) [inline middleware/middleware-quarkus/monitoring/01-install-prometheus.md:68]: Further down, the text says:

"You should see `replication controller "prometheus-2" successfully rolled out`"

But I see

`deployment "prometheus" successfully rolled out`

**Review by edeandrea — COMMENTED** [review]
@jamesfalkner Comments re: the monitoring scenario.

File: middleware/middleware-quarkus/panache/01-define-entity.md [hunk 8]
```diff
@@ -36,7 +36,7 @@ We need to add a few extensions to the app for Panache and Postgres. We'll use t
 
 Click this command to add the Hibernate ORM with Panache extension:
 
-`mvn -q quarkus:add-extension -Dextensions="hibernate-orm-panache"`{{execute}}
+`mvn quarkus:add-extension -Dextensions="hibernate-orm-panache"`{{execute}}
```
Comment (edeandrea) [inline middleware/middleware-quarkus/panache/01-define-entity.md:39]: Why not add both extensions in one shot?

Click this command to add the Hibernate ORM with Panache and PostgreSQL JDBC extensions:
 `mvn quarkus:add-extension -Dextensions="hibernate-orm-panache, jdbc-postgresql"`{{execute}}
 
You should see:

```
[SUCCESS] ✅  Extension io.quarkus:quarkus-hibernate-orm-panache has been installed
[SUCCESS] ✅  Extension io.quarkus:quarkus-jdbc-postgresql has been installed
```

File: middleware/middleware-quarkus/panache/02-remote-dev.md [hunk 9]
```diff
@@ -62,7 +62,7 @@ Quarkus offers the ability to automatically generate OpenShift resources based o
 
 Click the following command to add it to our project:
 
-`mvn -q quarkus:add-extension -Dextensions="openshift"`{{execute}}
+`mvn quarkus:add-extension -Dextensions="openshift"`{{execute}}
 
 You will see:
```
Comment (edeandrea) [inline middleware/middleware-quarkus/panache/02-remote-dev.md:68]: In the "Deploy application to OpenShift" section, the deployment variable

`-Dquarkus.openshift.env-vars.quarkus-launch-devmode.value=true`

should instead be

`-Dquarkus.openshift.env.vars.quarkus-launch-devmode=true`

The explanation underneath should be fixed too to match.

Comment (edeandrea) [inline middleware/middleware-quarkus/panache/02-remote-dev.md:68]: Additionally in the "Deploy application to OpenShift section", the instructions say the output of `curl -s http://people-quarkus.2886795283-80-elsy07.environments.katacoda.com/person | jq` should be

```json
[
  {
    "persistent": true,
    "id": 1,
    "birth": "1974-08-15",
    "eyes": "BLUE",
    "name": "Farid Ulyanov"
  },
  {
    "persistent": true,
    "id": 2,
    "birth": "1984-05-24",
    "eyes": "BROWN",
    "name": "Salvador L. Witcher"
  },
  {
    "persistent": true,
    "id": 3,
    "birth": "1999-04-25",
    "eyes": "HAZEL",
    "name": "Huỳnh Kim Huê"
  }
]

but the actual output is

```json
[
  {
    "id": 1,
    "birth": "1974-08-15",
    "eyes": "BLUE",
    "name": "Farid Ulyanov"
  },
  {
    "id": 2,
    "birth": "1984-05-24",
    "eyes": "BROWN",
    "name": "Salvador L. Witcher"
  },
  {
    "id": 3,
    "birth": "1999-04-25",
    "eyes": "HAZEL",
    "name": "Huỳnh Kim Huê"
  }
]
```

Also the text immediately after this says

```It's working! We'll leave it running and use Quarkus' Remote Live Reload feature to automatically update our app as we make changes. Note that the id and persistent fields were added to our entity, but never appear in our query APIs and can be safely ignored most of the time.```

But nowhere in the scenario did we ever update anything (nor have we even launched the `quarkus:remote-dev` goal yet. It seems out of place.

I think maybe the following "Remote Dev Connection" section should come before this section? Or at least before this `curl`?

**Review by edeandrea — COMMENTED** [review]
@jamesfalkner Comments re: the panache scenario

File: middleware/middleware-quarkus/panache/02-remote-dev.md (hunk 9 above)
Comment (jamesfalkner) [reply middleware/middleware-quarkus/panache/02-remote-dev.md:68]: reworded to be more clear that updates will be made soon.

Comment (jamesfalkner) [reply middleware/middleware-quarkus/panache/02-remote-dev.md:68]: also fixed the "you should see" output

Comment (jamesfalkner) [reply middleware/middleware-quarkus/panache/02-remote-dev.md:68]: fixed

File: middleware/middleware-quarkus/panache/01-define-entity.md (hunk 8 above)
Comment (jamesfalkner) [reply middleware/middleware-quarkus/panache/01-define-entity.md]: fixed

File: middleware/middleware-quarkus/monitoring/01-install-prometheus.md (hunk 7 above)
Comment (jamesfalkner) [reply middleware/middleware-quarkus/monitoring/01-install-prometheus.md:68]: fixed

File: middleware/middleware-quarkus/kafka/04-create-resource.md (hunk 6 above)
Comment (jamesfalkner) [reply middleware/middleware-quarkus/kafka/04-create-resource.md]: fixed

File: middleware/middleware-quarkus/kafka/04-create-resource.md (hunk 5 above)
Comment (jamesfalkner) [reply middleware/middleware-quarkus/kafka/04-create-resource.md]: fixed

File: middleware/middleware-quarkus/kafka/04-create-resource.md (hunk 4 above)
Comment (jamesfalkner) [reply middleware/middleware-quarkus/kafka/04-create-resource.md]: fixed

File: middleware/middleware-quarkus/getting-started/03-package-and-run.md (hunk 3 above)
Comment (jamesfalkner) [reply middleware/middleware-quarkus/getting-started/03-package-and-run.md]: fixed

File: middleware/middleware-quarkus/getting-started/01-create-project.md (hunk 2 above)
Comment (jamesfalkner) [reply middleware/middleware-quarkus/getting-started/01-create-project.md]: fixed

File: middleware/middleware-quarkus/getting-started/01-create-project.md (hunk 1 above)
Comment (jamesfalkner) [reply middleware/middleware-quarkus/getting-started/01-create-project.md:93]: Awaiting expert to weigh in..

File: middleware/middleware-quarkus/qute/03-parameters.md [hunk 10]
```diff
@@ -46,10 +46,10 @@ Create a Qute template at by clicking `qute/src/main/resources/templates/ItemRes
...
-And create a simple `ItemService` to mock up a database of items. Click `qute/src/main/java/org/acme/resteasyqute/ItemService.java`{{open}} to open the file, then click **Copy To Editor**:
+And create a simple `ItemService` to mock up a database of items. Click `qute/src/main/java/org/acme/ItemService.java`{{open}} to open the file, then click **Copy To Editor**:
 
-<pre class="file" data-filename="./qute/src/main/java/org/acme/resteasyqute/ItemService.java" data-target="replace">
-package org.acme.qute;
+<pre class="file" data-filename="./qute/src/main/java/org/acme/ItemService.java" data-target="replace">
+package org.acme;
 
 import java.math.BigDecimal;
 import java.util.HashMap;
```
Comment (edeandrea) [inline middleware/middleware-quarkus/qute/03-parameters.md:55]: The current

```java
private Map<Integer, Item> items = new HashMap<Integer, Item>() {{
        put(1, new Item(new BigDecimal(1.99), "Apple"));
        put(2, new Item(new BigDecimal(2.99), "Pear"));
        put(3, new Item(new BigDecimal(3.99), "Grape"));
        put(4, new Item(new BigDecimal(129.99), "Mango"));
    }};
```

could be re-written in a cleaner way as

```java
private Map<Integer, Item> items = Map.of(
      1, new Item(new BigDecimal(1.99), "Apple"),
      2, new Item(new BigDecimal(2.99), "Pear"),
      3, new Item(new BigDecimal(3.99), "Grape"),
      4, new Item(new BigDecimal(129.99), "Mango")
);
```

File: middleware/middleware-quarkus/qute/04-extensions.md [hunk 11]
```diff
@@ -27,12 +27,12 @@ Notice the use of Handlebar-esque `{#if ...}`. It's part of Qute's basic control
...
 
 Click **Copy to Editor** to create the extensions class where we'll declare our `discountedPrice`:
 
-<pre class="file" data-filename="./qute/src/main/java/org/acme/resteasyqute/TemplateExtensions.java" data-target="replace">
-package org.acme.qute;
+<pre class="file" data-filename="./qute/src/main/java/org/acme/TemplateExtensions.java" data-target="replace">
+package org.acme;
 
 import java.math.BigDecimal;
 import io.quarkus.qute.TemplateExtension;
```
Comment (edeandrea) [inline middleware/middleware-quarkus/qute/04-extensions.md:38]: There's a typo in the paragraph directly after this code listing:

```
Here we declare a static template extension method that can be used to add "computed properties" to a data class. The class of the first parameter (in this case Item) is used to match the base object and the method name is used to match the property name (in this case discountedPrice). When we declare {item.computedPrice}, the contextual value of item is passed to this discountedPrice method to compute its value (where we use the item.price.multiply() method to apply a 10% discount by multiplying by 0.9).
```

It references `{item.computedPrice}`. This should be `{item.discountedPrice}`. The paragraph should be:

```
Here we declare a static template extension method that can be used to add "computed properties" to a data class. The class of the first parameter (in this case Item) is used to match the base object and the method name is used to match the property name (in this case discountedPrice). When we declare {item.discountedPrice}, the contextual value of item is passed to this discountedPrice method to compute its value (where we use the item.price.multiply() method to apply a 10% discount by multiplying by 0.9).
```

File: middleware/middleware-quarkus/qute/05-periodic.md [hunk 12]
```diff
@@ -6,10 +6,10 @@ Templating engine could be also very useful when rendering periodic reports. We
```
Comment (edeandrea) [inline middleware/middleware-quarkus/qute/05-periodic.md:6]: The first sentence starts with `Templating engine could `. I think it should be `Templating engines can `

File: middleware/middleware-quarkus/qute/05-periodic.md [hunk 13]
```diff
@@ -6,10 +6,10 @@ Templating engine could be also very useful when rendering periodic reports. We
...
 
-Click `qute/src/main/java/org/acme/resteasyqute/Sample.java`{{open}} to open the file. Click **Copy to Editor** to paste in the code:
+Click `qute/src/main/java/org/acme/Sample.java`{{open}} to open the file. Click **Copy to Editor** to paste in the code:
 
-<pre class="file" data-filename="./qute/src/main/java/org/acme/resteasyqute/Sample.java" data-target="replace">
-package org.acme.qute;
+<pre class="file" data-filename="./qute/src/main/java/org/acme/Sample.java" data-target="replace">
+package org.acme;
 
 public class Sample {
```
Comment (edeandrea) [inline middleware/middleware-quarkus/qute/05-periodic.md:14]: I know you didn't make this change, but `public` attributes just bother me :)

File: middleware/middleware-quarkus/qute/05-periodic.md [hunk 14]
```diff
@@ -25,10 +25,10 @@ public class Sample {
 }
 </pre>
 
-Now let's ceate a service whose `get()` method returns a random list of samples. Open the file with `qute/src/main/java/org/acme/resteasyqute/SampleService.java`{{open}} and click **Copy to Editor** to create the code:
+Now let's ceate a service whose `get()` method returns a random list of samples. Open the file with `qute/src/main/java/org/acme/SampleService.java`{{open}} and click **Copy to Editor** to create the code:
 
-<pre class="file" data-filename="./qute/src/main/java/org/acme/resteasyqute/SampleService.java" data-target="replace">
-package org.acme.qute;
+<pre class="file" data-filename="./qute/src/main/java/org/acme/SampleService.java" data-target="replace">
```
Comment (edeandrea) [inline middleware/middleware-quarkus/qute/05-periodic.md:30]: The `SampleService.java` class here is old & ugly, written pre-Java 8. This would be much nicer:

```java
package org.acme;

import java.util.List;
import java.util.Random;
import java.util.stream.Collectors;
import java.util.stream.IntStream;

import javax.enterprise.context.ApplicationScoped;

@ApplicationScoped
public class SampleService {
    private static final String[] names = {"James", "Deepak", "Daniel", "Shaaf", "Jeff", "Sally"};

    public List<Sample> get() {
        int count = new Random().nextInt(10);
        return IntStream.range(0, count)
            .mapToObj(idx -> Math.random() > 0.5)
            .map(valid -> new Sample(valid, names[(int)(Math.random() * names.length)], Math.random() + ""))
            .collect(Collectors.toList());
    }
}
```

File: middleware/middleware-quarkus/qute/06-reactive.md [hunk 15]
```diff
@@ -12,18 +12,17 @@ Quarkus [Reactive routes](https://quarkus.io/guides/reactive-routes) propose an
```
Comment (edeandrea) [inline middleware/middleware-quarkus/qute/06-reactive.md:12]: This section uses reactive routes. Do we want to keep it this way or use the reactive qute extension with resteasy reactive?

Personally I think showcasing RESTEasy Reactive (meaning building a resource class with `Uni`/`Multi` would be better.

But I get it - minimal amount of change to get to Quarkus 2. This would involve re-writing this entire guide for this section.

**Review by edeandrea — COMMENTED** [review]
@jamesfalkner Comments re: the Qute scenario

File: middleware/middleware-quarkus/reactive-sql/04-add-model.md [hunk 16]
```diff
@@ -35,7 +35,7 @@ In the above you can see we use a `PgPool` object and once the query is prepared
```
Comment (edeandrea) [inline middleware/middleware-quarkus/reactive-sql/04-add-model.md:35]: In the step 03, If we update to RESTEasy Reactive, the code snippets need to change `@PathParam` to `@PathParam("id")`

Comment (edeandrea) [reply middleware/middleware-quarkus/reactive-sql/04-add-model.md:35]: This could be done regardless and also remove the `import org.jboss.resteasy.annotations.jaxrs.PathParam;` from `CoffeeResource`. This would then use the JAX-RS `@PathParam` annotation instead of the RESTEasy-specific one, which accomplishes the same thing.

**Review by edeandrea — COMMENTED** [review]
@jamesfalkner Comments re: the reactive-sql project

File: middleware/middleware-quarkus/qute/01-basics.md [hunk 17]
```diff
@@ -52,11 +52,13 @@ __  ____  __  _____   ___  __ ____  ______
 
 Note the amazingly fast startup time! The app is now running "locally" (within the Linux container in which this exercise runs).
 
-Test that the app is running by accessing the sample app [using this link](https://[[CLIENT_SUBDOMAIN]]-8080-[[KATACODA_HOST]].environments.katacoda.com/qute/quarks). You should see
+Test that the app is running by accessing the sample page [using this link](https://[[CLIENT_SUBDOMAIN]]-8080-[[KATACODA_HOST]].environments.katacoda.com/some-page). You should see
 
 ![Qute sample](/openshift/assets/middleware/quarkus/qute-sample.png)
```
[the remark this answers is not in the document]
Comment (jamesfalkner) [reply middleware/middleware-quarkus/qute/01-basics.md:57]: Yeah Ive update the image, but Katacoda test env doesnt show updated images until they go to prod..

File: middleware/middleware-quarkus/qute/01-basics.md [hunk 18]
```diff
@@ -52,11 +52,13 @@ __  ____  __  _____   ___  __ ____  ______
...
+This page is rendered using the `qute/src/main/resources/templates/page.qute.html`{{open}} HTML template. If you look closely you can see a `{name ?: "Qute"}` directive that renders the passed-in `name` query present in the `qute/src/main/java/org/acme/SomePage.java` class and passed into the template renderer in the `data()` method, making the `name` variable available to the renderer and returning the rendered HTML to your browser. If you pass a different name, say, `Jerry` using [this link](https://[[CLIENT_SUBDOMAIN]]-8080-[[KATACODA_HOST]].environments.katacoda.com/some-page?name=Jerry) (which adds `?name=Jerry` to the URL) you'll see the new name rendered via the Qute template:
+
+![Qute sample](/openshift/assets/middleware/quarkus/qute-sample-jerry.png)
```
[the remark this answers is not in the document]
Comment (jamesfalkner) [reply middleware/middleware-quarkus/qute/01-basics.md:61]: yep same deal with katacoda test vs. prod. I did update it to the new image: https://github.com/jamesfalkner/learn-katacoda/blob/quarkus2update/assets/middleware/quarkus/qute-sample-jerry.png

File: middleware/middleware-quarkus/qute/06-reactive.md (hunk 15 above)
Comment (jamesfalkner) [reply middleware/middleware-quarkus/qute/06-reactive.md:12]: agreed, better to leave this for a future PR.

File: middleware/middleware-quarkus/qute/05-periodic.md (hunk 13 above)
Comment (jamesfalkner) [reply middleware/middleware-quarkus/qute/05-periodic.md:14]: I think they have to be public to be referenced from templates.. See https://quarkus.io/guides/qute-reference#template_data _ Non-public members, constructors, static initializers, static, synthetic and void methods are always ignored._

File: middleware/middleware-quarkus/spring/07-deploy-and-test.md [hunk 19]
```diff
@@ -8,7 +8,6 @@ Run the following command to add it to our project:
```
Comment (edeandrea) [inline middleware/middleware-quarkus/spring/07-deploy-and-test.md:8]: Should this be renamed to `06-deploy-and-test.md` since its now step 6 in the scenario?

File: middleware/middleware-quarkus/spring/07-deploy-and-test.md [hunk 20]
```diff
@@ -135,13 +131,13 @@ With that set, let's see how fast our app can scale up to 10 instances:
 
 Back in the [OpenShift Developer Toplogy](https://console-openshift-console-[[HOST_SUBDOMAIN]]-443-[[KATACODA_HOST]].environments.katacoda.com/topology/ns/quarkus-spring) you'll see the app scaling dynamically up to 10 pods:
 
-![Scaling](/openshift/assets/middleware/quarkus/scaling_spring_.png)
+![Scaling](/openshift/assets/middleware/quarkus/scaling_spring.png)
```
Comment (edeandrea) [inline middleware/middleware-quarkus/spring/07-deploy-and-test.md:134]: I see the name change to the file (`scaling_spring_.png` -> `scaling_spring.png`) but I don't see the file rename as part of the commit?

File: middleware/middleware-quarkus/spring/07-deploy-and-test.md [hunk 21]
```diff
@@ -135,13 +131,13 @@ With that set, let's see how fast our app can scale up to 10 instances:
...
 
 We now have 10 instances running providing better performance. Make sure it still works:
 
 `curl -s http://fruit-taster-quarkus-spring.[[HOST_SUBDOMAIN]]-80-[[KATACODA_HOST]].environments.katacoda.com/taster | jq`{{execute T1}}
 
-**10 not enough? Try 100!** Click the command to scale this app to 100 instances:
+**10 not enough? Try 50!** Click the command to scale this app to 100 instances:
 
 `oc scale --replicas=50 dc/fruit-taster`{{execute T1}}
```
Comment (edeandrea) [inline middleware/middleware-quarkus/spring/07-deploy-and-test.md:143]: The part

You can watch the 50 pods spinning up:
`oc get pods -w -l app=fruit-taster`

Doesn't ever show anything. The deployment doesn't have a label `app=fruit-taster`, so it never finds anything.

**Review by edeandrea — COMMENTED** [review]
@jamesfalkner Comments re: spring scenario

File: middleware/middleware-quarkus/spring/07-deploy-and-test.md [hunk 22]
```diff
@@ -135,13 +131,13 @@ With that set, let's see how fast our app can scale up to 10 instances:
...
 
-![Scaling](/openshift/assets/middleware/quarkus/scaling_spring_.png)
+![Scaling](/openshift/assets/middleware/quarkus/scaling_spring.png)
 
 We now have 10 instances running providing better performance. Make sure it still works:
 
 `curl -s http://fruit-taster-quarkus-spring.[[HOST_SUBDOMAIN]]-80-[[KATACODA_HOST]].environments.katacoda.com/taster | jq`{{execute T1}}
 
-**10 not enough? Try 100!** Click the command to scale this app to 100 instances:
+**10 not enough? Try 50!** Click the command to scale this app to 100 instances:
```
Comment (edeandrea) [inline middleware/middleware-quarkus/spring/07-deploy-and-test.md:140]: You say "Try 50!" but then it says "Click the command to scale this app to 100 instances". Shouldn't it be 50 instances?

**Comment by edeandrea** [comment]
@jamesfalkner I've completed the review.

File: middleware/middleware-quarkus/getting-started/01-create-project.md (hunk 1 above)
Comment (edeandrea) [reply middleware/middleware-quarkus/getting-started/01-create-project.md:93]: We can leave it alone for now. I don't think its an issue.

File: middleware/middleware-quarkus/qute/05-periodic.md (hunk 13 above)
Comment (edeandrea) [reply middleware/middleware-quarkus/qute/05-periodic.md:14]: Thats what it seems.

File: middleware/middleware-quarkus/spring/07-deploy-and-test.md (hunk 20 above)
Comment (jamesfalkner) [reply middleware/middleware-quarkus/spring/07-deploy-and-test.md:134]: The file was always called `scaling_spring.png`, just the instructions had the wrong name in the markdown. Here's the file (in master): https://github.com/openshift-labs/learn-katacoda/blob/master/assets/middleware/quarkus/scaling_spring.png

Comment (edeandrea) [reply middleware/middleware-quarkus/spring/07-deploy-and-test.md:134]: Ahhhhh nevermind then!

File: middleware/middleware-quarkus/reactive-sql/02-remote-dev.md [hunk 23]
```diff
@@ -62,7 +62,7 @@ Quarkus offers the ability to automatically generate OpenShift resources based o
 
 Click the following command to add it to our project:
 
-`mvn -q quarkus:add-extension -Dextensions="openshift"`{{execute}}
+`mvn quarkus:add-extension -Dextensions="openshift"`{{execute}}
 
 You will see:
```
[the remark this answers is not in the document]
Comment (jamesfalkner) [reply middleware/middleware-quarkus/reactive-sql/02-remote-dev.md:68]: yeah and i'll give it a better title too

File: middleware/middleware-quarkus/qute/03-parameters.md (hunk 10 above)
Comment (jamesfalkner) [reply middleware/middleware-quarkus/qute/03-parameters.md:55]: fixed

File: middleware/middleware-quarkus/qute/04-extensions.md (hunk 11 above)
Comment (jamesfalkner) [reply middleware/middleware-quarkus/qute/04-extensions.md:38]: fixed

File: middleware/middleware-quarkus/qute/05-periodic.md (hunk 12 above)
Comment (jamesfalkner) [reply middleware/middleware-quarkus/qute/05-periodic.md:6]: fixed

File: middleware/middleware-quarkus/qute/05-periodic.md (hunk 14 above)
Comment (jamesfalkner) [reply middleware/middleware-quarkus/qute/05-periodic.md:30]: fixed

File: middleware/middleware-quarkus/reactive-sql/02-remote-dev.md (hunk 23 above)
[the remark this answers is not in the document]
Comment (jamesfalkner) [reply middleware/middleware-quarkus/reactive-sql/02-remote-dev.md:68]: fixed

[the remark this answers is not in the document]
Comment (jamesfalkner) [reply middleware/middleware-quarkus/reactive-sql/02-remote-dev.md:68]: fixed

File: middleware/middleware-quarkus/reactive-sql/04-add-model.md (hunk 16 above)
Comment (jamesfalkner) [reply middleware/middleware-quarkus/reactive-sql/04-add-model.md:35]: fixed (also fixed import statement in https://github.com/openshift-katacoda/rhoar-getting-started/pull/78/commits/7a8fb5de419b4a269af8ba7c1778fbfa90c9003a )

File: middleware/middleware-quarkus/spring/07-deploy-and-test.md (hunk 19 above)
Comment (jamesfalkner) [reply middleware/middleware-quarkus/spring/07-deploy-and-test.md:8]: fixed

File: middleware/middleware-quarkus/spring/07-deploy-and-test.md (hunk 21 above)
Comment (jamesfalkner) [reply middleware/middleware-quarkus/spring/07-deploy-and-test.md:143]: fixed

File: middleware/middleware-quarkus/spring/07-deploy-and-test.md (hunk 22 above)
Comment (jamesfalkner) [reply middleware/middleware-quarkus/spring/07-deploy-and-test.md]: fixed

File: middleware/middleware-quarkus/qute/02-typesafe.md [hunk 24]
```diff
@@ -71,10 +72,10 @@ Keep in mind this style of declaration allows you to reference templates declare
...
-Click to open `qute/src/main/java/org/acme/resteasyqute/GoodbyeResource.java`{{open}} then click **Copy to Editor** to create a new resource:
+Click to open `qute/src/main/java/org/acme/GoodbyeResource.java`{{open}} then click **Copy to Editor** to create a new resource:
 
-<pre class="file" data-filename="./qute/src/main/java/org/acme/resteasyqute/GoodbyeResource.java" data-target="replace">
-package org.acme.qute;
+<pre class="file" data-filename="./qute/src/main/java/org/acme/GoodbyeResource.java" data-target="replace">
+package org.acme;
 
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
```
[the remark this answers is not in the document]
Comment (jamesfalkner) [reply middleware/middleware-quarkus/qute/02-typesafe.md:81]: Taken from the guides - will file a bug against them and update if and when its fixed.
